data/test/demo-rasa-composite-entities.yml

```diff
@@ -13,26 +13,6 @@ nlu:
     - correct
     - great choice
     - sounds really good
-- intent: goodbye
-  examples: |
-    - bye
-    - goodbye
-    - good bye
-    - stop
-    - end
-    - farewell
-    - Bye bye
-    - have a good one
-- intent: greet
-  examples: |
-    - hey
-    - howdy
-    - hey there
-    - hello
-    - hi
-    - good morning
-    - good evening
-    - dear sir
```

Comment:
Again, attempt to bring test memory consumption down so that runners can handle it.